kview/src/main/java/dev/ikm/komet/kview/mvvm/model/ViewCoordinateHelper.java

```diff
@@ -55,6 +58,20 @@ public static ViewCalculatorWithCache createViewCalculatorLatestByTime(ViewPrope
         return new ViewCalculatorWithCache(latestViewCoordinate);
     }
 
+
+    /**
+     * Create a new View Calculator with stated navigation with an updated view coordinate & view calculator.
+     * @param viewProperties A given view property
+     * @return newViewCalculatorWithCache
+     */
+    public static ViewCalculatorWithCache createNavigationCalculatorLatest(ViewProperties viewProperties) {
+        ViewCalculator existingViewCalculator = viewProperties.calculator();
+        IntIdSet intIdSet = existingViewCalculator.navigationCoordinate().navigationPatternNids().with(TinkarTerm.STATED_NAVIGATION_PATTERN.nid());
```

Comment:
Can you add another parameter to allow the caller to specify the `navigationPatternNids`?
If you add, can the method name make sense to the caller. `createNavigationCalculatorWithPatternNids(ViewProperties viewProperties, IntIdSet)`

This way the caller can be more specific. Please update the comments to reflect the method.